internal/archive/archive_test.go

```diff
@@ -409,44 +436,122 @@ func read(r io.Reader) string {
 
 // ----------------------------------------------------------------------------------------
 // Real archive tests, only enabled via --real-archive.
+//
+// Additionally, run tests against Ubuntu Pro archives by passing --pro-archive.
+// The host machine must be Pro enabled and relevant Pro services must be
+// enabled. The following commands might help:
+// 		sudo pro attach <pro-token> --no-auto-enable
+//		sudo pro enable fips fips-updates esm-apps esm-infra --assume-yes
```

Comment:
`fips` and `fips-updates` cannot be enabled at the same time. Running this command will have
```
Disabling incompatible service: FIPS
Removing APT access to FIPS
```

Reply:
TODO: add a comment saying although disabled, this command writes the credentials of each service/endpoint.